### PR TITLE
Remove lodging from trip

### PR DESCRIPTION
**Pull request by Omniforce**
- Added button to remove lodging from trip
- Shows currently selected lodging on trip-view view

### CODE REVIEW

**Comment by alfred** [comment]
Why are there duplicate commits? Why did you merge your own branch into itself?

File: public/views/lodging/lodging-add/lodging-add.controller.client.js [hunk 1]
```diff
@@ -23,6 +24,15 @@
         });
     }
 
+    function removeLodging() {
+      vm.trip.lodging = null;
```
Comment (alfred) [inline public/views/lodging/lodging-add/lodging-add.controller.client.js]: `delete vm.trip.lodging;`

Comment (alfred) [inline public/views/lodging/lodging-add/lodging-add.controller.client.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete

Comment (Omniforce) [inline public/views/lodging/lodging-add/lodging-add.controller.client.js]: so I think if you just do `delete ...`, it doesn't update that server side. It just ignores the field

Comment (alfred) [inline public/views/lodging/lodging-add/lodging-add.controller.client.js]: Oh rip wow I hate that

**Pull request merged**